modules/tls-profiles-kubelet-configuring.adoc

```diff
@@ -0,0 +1,121 @@
+= Configuring TLS profiles for the Kubelet
+
+To configure a TLS security profile for the kubelet when it is acting as an HTTP server, create a `KubeletConfig` custom resource (CR) to specify a predefined or custom TLS profile for specific nodes. If a TLS profile is not configured, the default TLS profile is `Intermediate`.
+
+.Sample `KubeletConfig` CR that configures the `Old` TLS security profile on worker nodes
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: KubeletConfig
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: ""
+ ...
+----
+
+The kubelet uses its HTTP/GRPC server to communicate with the Kubernetes API server, which sends commands to pods, gathers logs, and run exec commands on pods through the kubelet. The TLS profile defines the minimum TLS version and the TLS ciphers that the Kubernetes API server must use when connecting with the kubelet to protect communication between the kubelet and the Kubernetes API server. By default, when the kubelet acts as a client with the Kubernetes API server, it automatically negotiates the TLS parameters with the API server.
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile in the `kubelet.conf` file on a configured node.
+
+[IMPORTANT]
+====
+The kubelet does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3` it is not supported. The kubelet converts the `Modern` profile to `Intermediate`.
+
+The kubelet also converts the TLS `1.0` of an `Old` or `Custom` profile to `1.1`, and TLS `1.3` of a `Custom` profile to `1.2`.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a `KubeletConfig` CR to configure the TLS security profile:
++
+.Sample `KubeletConfig` CR for a `Custom` profile
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: set-kubelet-tls-security-profile
+spec:
+  tlsSecurityProfile:
+    type: Custom <1>
+    custom: <2>
+      ciphers: <3>
+      - ECDHE-ECDSA-CHACHA20-POLY1305
+      - ECDHE-RSA-CHACHA20-POLY1305
+      - ECDHE-RSA-AES128-GCM-SHA256
+      - ECDHE-ECDSA-AES128-GCM-SHA256
+      minTLSVersion: VersionTLS11
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <3>
+----
++
+<1> Specify the TLS security profile type. The default is `Intermediate`.
+* `Old`
+* `Intermediate`
+// * `Modern`
+* `Custom`
+<2> Specify:
+* `Old: {}`
+* `Intermediate: {}`
+// * `Modern: {}`
+* `Custom:`
+<3> For the `custom` type, specify a list of TLS ciphers and minimum accepted TLS version.
+
+. Create the `KubeletConfig` object:
++
+----
+$ oc create -f <file-name>
+----
+
+.Verification
+
+. After the nodes are in the `Ready` state, verify that the profile is set on a configured node:
```

Comment:
Instead of using a single step, could you make each of these commands its own step with a sentence? I know that there are some existing procedures you could find that already have wording for these commands, like: https://docs.openshift.com/container-platform/4.7/backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-crashlooping-etcd-member_replacing-unhealthy-etcd-member